src/Api/V1/Tests/Action/CoreTest.php

```diff
@@ -0,0 +1,237 @@
+<?php
+
+namespace Api\V1\Tests\Action;
+
+use Common\ApiTestCase;
+
+class CoreTest extends ApiTestCase
+{
+    public function testErrorOutput()
```

Comment:
Absolutely correct, I gave all methods a more verbose name in 2424b667b65cdbf19608ad3aba9a3da9706a7072